src/content/api/cli.mdx

```diff
@@ -596,15 +600,15 @@ module.exports = (env, argv) => {
 };
 ```
 
-### node-env
+### define-process-env-node-env
```

Comment:
I think it's better to keep both versions. Most users might not realize it's webpack-cli v5+ only, and it will take a long time for users to upgrade to webpack-cli v5.

E.g.,

```
### node-env

for `webpack-cli v4`

### define-process-env-node-env

for `webpack-cli v5+`
```

Reply:
Makes sense 👍